### PR TITLE
speed up absorb

### PR DESCRIPTION
**Pull request by evanmcc**
we're doing a lot of silly stuff that's slowing absorb down.
- we don't use score anymore but we continue to update it
- we validate witnesses that we're not going to score

this PR:
- clears all scores (for snapshot ledger convergence) and stops updating them
- removes witness validation during absorb until we can land #919

### CODE REVIEW

File: src/transactions/blockchain_txn.erl [hunk 1]
```diff
@@ -433,16 +433,20 @@ absorb_and_commit(Block, Chain0, BeforeCommit, Rescue) ->
...
                 {ok, Chain2} ->
                     Ledger2 = blockchain:ledger(Chain2),
                     Hash = blockchain_block:hash_block(Block),
                     case BeforeCommit(Chain2, Hash) of
                         ok ->
                             ok = blockchain_ledger_v1:commit_context(Ledger2),
-                            absorb_delayed(Block, Chain0),
-                            absorb_aux(Block, Chain0);
+                            End2 = erlang:monotonic_time(millisecond),
+                            ok = absorb_delayed(Block, Chain0),
```
Comment (Vagabond) [inline src/transactions/blockchain_txn.erl:444]: Is the goal to crash if these secondary absorbs fail?

**Review by Vagabond — COMMENTED** [review]
Overall this looks good, but I'm not sure about asserting the secondary absorbs succeed, especially for aux absorb.

File: src/transactions/blockchain_txn.erl [hunk 2]
```diff
@@ -433,16 +433,20 @@ absorb_and_commit(Block, Chain0, BeforeCommit, Rescue) ->
...
+                            ok = absorb_aux(Block, Chain0),
+                            End3 = erlang:monotonic_time(millisecond),
+                            lager:info("validation took ~p absorb took ~p post took ~p ms height ~p",
```
Comment (abhay) [inline src/transactions/blockchain_txn.erl:447]:
```suggestion
                            lager:info("validation took ~p absorb took ~p post took ~p ms for block height ~p",
```

Comment (abhay) [reply src/transactions/blockchain_txn.erl:447]: optional

**Review by Vagabond — APPROVED** [review]

File: src/transactions/blockchain_txn.erl (hunk 2 above)
Comment (BenoitDuffez) [reply src/transactions/blockchain_txn.erl]: `lager:debug` perhaps? the traces are very verbose

Comment (evanmcc) [reply src/transactions/blockchain_txn.erl]: you can tune `blockchain` `slow_txn_log_threshold` with the .1 release for the low txn logging.  this is one line per block, which I think is reasonable.

**Pull request merged**